charts/dap-secret-webhook/templates/tls-job.yaml

```diff
@@ -0,0 +1,117 @@
+{{- if .Values.tls.create -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+    name: {{ include "dap-secret-webhook.name" . }}-tls-sa
+    namespace: {{ .Release.Namespace }}
+    annotations:
+      "helm.sh/hook": pre-install, pre-upgrade
+      "helm.sh/hook-weight": "1"
+      "helm.sh/hook-delete-policy": before-hook-creation, hook-succeeded, hook-failed	
+---
+{{ end }}
+{{- if .Values.rbac.create -}}
```

Comment:
Do you think it would be good to create when tls.create is true?

Reply:
I think the end block is missing for this block. Can you confirm if it is intended to create the job resource on this condition